lib/ansible/modules/cloud/cloudstack/cs_affinitygroup.py

```diff
@@ -36,7 +36,7 @@
     description:
       - Name of the affinity group.
     required: true
-  affinty_type:
+  affinity_type:
     description:
       - Type of the affinity group. If not specified, first found affinity type is used.
     required: false
```

Comment:
Please add the old value as an alias here:

~~~diff
...
        - Type of the affinity group. If not specified, first found affinity type is used. 
+       - The misspelled C(affinty_type) is for backwards compatibility and will be removed in version 2.9.
    required: false
+   aliases: [ affinty_type ]